rero_ils/modules/items/api/record.py

```diff
@@ -218,7 +219,7 @@ def link_item_to_holding(cls, record, mode):
 
         if (
             mode == 'create' and record.get('holding')) or (
-            old_holding_type in ['serial', 'electronic']
+            old_holding_type in [HoldingTypes.SERIAL, HoldingTypes.ELECTRONIC]
```

Comment:
`Enum` class can be iterated.

Reply:
I can't enumerate on the Enum class itself because it contains 3 values ; Only 2 are valid in this test.